clang/lib/Format/BreakableToken.cpp

```diff
@@ -927,14 +927,12 @@ BreakableLineCommentSection::BreakableLineCommentSection(
       }
 
       if (Lines[i].size() != IndentPrefix.size()) {
-        PrefixSpaceChange[i] = FirstLineSpaceChange;
+        assert(Lines[i].size() > IndentPrefix.size());
 
-        if (SpacesInPrefix + PrefixSpaceChange[i] < Minimum) {
-          PrefixSpaceChange[i] +=
-              Minimum - (SpacesInPrefix + PrefixSpaceChange[i]);
-        }
+        PrefixSpaceChange[i] = SpacesInPrefix + FirstLineSpaceChange < Minimum
```

Comment:
```suggestion
        PrefixSpaceChange[i] = (SpacesInPrefix + FirstLineSpaceChange) < Minimum
```
This is what you wanted, or?

Reply:
Why would we want to add the redundant parentheses? They were not there in the `if` statement before.

Reply:
I see, but in general, how could `var1 + var2 < var3` be misread as `var1 + (var2 < var3)`, whether the former is in if/while, ternary, or assignment? In fact, I would always remove the redundant parentheses in `(var1 + var2) < var3`.

Reply:
Yeah, I was mistaken.

But, even is not needed, I'd do this:
```suggestion
        PrefixSpaceChange[i] = (SpacesInPrefix + FirstLineSpaceChange < Minimum)
```

Reply:
Like the examples of [conditional/ternary operators](https://en.cppreference.com/w/cpp/language/operator_other.html#Conditional_operator), I never add redundant parentheses because they have the lowest precedence relative to arithmetic/logical operators. This is analogous to `a + (b * c)` or `(a / b) - c`.

The clang-format codebase has a mixture of both `expr1 ? expr2 : expr3` and `(expr1) ? expr2 : expr3`, and IMO we should consistently use the former. Although I can understand why some people prefer the latter when `expr1` and `expr2` are on the same line, there's really nothing to gain when the questions mark starts on a new line:
```
expr1
    ? expr2
```